xml/Sector.xml

```diff
@@ -2549,7 +2789,7 @@
 		<codelist-item>
 			<code>91010</code>
 			<name>
-				<narrative>Administrative costs</narrative>
+				<narrative>Administrative costs (non-sector allocable)</narrative>
```

Comment:
Descriptions, you mean? That’s true, there’s no description here. But that’s unchanged (i.e. there was no description here before, either).